recipe/meta.yaml

```diff
@@ -20,18 +20,17 @@ requirements:
     - python                                 # [build_platform != target_platform]
   host:
     - python
-    - pip >=18
-    - setuptools >=41
+    - pip
+    - setuptools >=41.0.0
     - setuptools_scm >=2
+    - wheel >=0.30.0
   run:
     - python
-    - appdirs >=1.4.3,<2
     - distlib >=0.3.1,<1
-    - filelock >=3.0.0,<4
+    - filelock >=3.2,<4
     - importlib-metadata >=0.12  # [py<38]
     - importlib_resources >=1.0  # [py<37]
-    - pathlib2 >=2.3.3,<3  # [py<34 and not win]
-    - six >=1.9.0,<2   # keep it >=1.9.0 as it may cause problems on LTS platforms
+    - platformdirs >=2,<3
 
 test:
   imports:
```

Comment:
I think we can extend `test/imports` because the package has more modules:
```
    - virtualenv
    - virtualenv.activation
    - virtualenv.app_data
    - virtualenv.config
    - virtualenv.create
    - virtualenv.discovery
    - virtualenv.run
    - virtualenv.seed
    - virtualenv.util
```
see https://github.com/pypa/virtualenv/tree/20.16.5/src/virtualenv

Reply:
That would be a good addition.

Reply:
Thank you, I added the tests.